apps/ledger-live-desktop/src/renderer/families/cardano/DelegationFlowModal/steps/StepSummary.tsx

```diff
@@ -20,114 +21,114 @@ const Separator = styled.div`
   margin: 15px 0;
 `;
 
-export default class StepSummary extends PureComponent<StepProps> {
```

Comment:
We also loose the PureComponent here with no memo, maybe not as important as the row components